crates/bevy_color/Cargo.toml

```diff
@@ -10,17 +10,18 @@ keywords = ["bevy", "color"]
 rust-version = "1.76.0"
 
 [dependencies]
-bevy_math = { path = "../bevy_math", version = "0.14.0-dev" }
+bevy_math = { path = "../bevy_math", version = "0.14.0-dev", default-features = false }
```

Comment:
if we are disabling the default-features here, shouldn't we enable bevy_math/bevy_reflect when the user enables bevy_reflect?

Reply:
Honestly I'm not quite sure what's most correct.

It sounds like that might be convenient.
On the other hand the user can always choose to include `bevy_math` in their deps directly and choose to have the reflect feature there.

Reply:
My only concern is that some problem can happen if we try to reflect something from bevy_math without the reflect support, but I guess this is something that will not happen, at least for now.